src/lib/containers/entity/SchemaDrivenAnnotationEditor/field/AdditionalPropertiesSchemaField.tsx

```diff
@@ -158,19 +149,8 @@ export function AdditionalPropertiesSchemaField<T>(
     AdditionalPropertyWidget.TextWidget,
   )
 
-  useEffect(() => {
-    // The item may not be an array when we get it, and we need to convert it right away because the order of items is not stable, and seems to depend on if the item is an array or not.
-    // Otherwise, the order of the properties will change when the user modifies the data. We may be able to fix this by modifying react-jsonschema-form to stabilize the item order.
-
-    // FIXME: This doesn't work without a delay.
-    setTimeout(() => {
-      onChange(list)
-    }, 50)
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
```

Comment:
This logic was moved to ObjectField via AnnotationEditorUtils.objectFieldComponentDidUpdate